src/nvim/testdir/test_usercommands.vim

```diff
@@ -360,7 +360,7 @@ func Test_addr_all()
       break
     endif
   endfor
-  call assert_equal(low, g:a1)
+  " call assert_equal(low, g:a1)
```

Comment:
I looked through the patch and found no obvious typo.

Then I added this to the test:

```diff
diff --git i/src/nvim/testdir/test_usercommands.vim w/src/nvim/testdir/test_usercommands.vim
index 841c72382..ebadf8d8d 100644
--- i/src/nvim/testdir/test_usercommands.vim
+++ w/src/nvim/testdir/test_usercommands.vim
@@ -360,6 +360,9 @@ func Test_addr_all()
       break
     endif
   endfor
+  call assert_equal(filter(range(1, bufnr('$')), 'buflisted(v:val)'), 999)
+  call assert_equal(filter(range(1, bufnr('$')), 'bufloaded(v:val)'), 999)
+  call assert_equal(g:a1, 999)
   " call assert_equal(low, g:a1)
   call assert_equal(bufnr('$'), g:a2)
```

which resulted in:

```
        function RunTheTest[37]..Test_addr_all line 19: Expected [3, 4, 5] but got 999
        function RunTheTest[37]..Test_addr_all line 20: Expected [3] but got 999
        function RunTheTest[37]..Test_addr_all line 21: Expected 2 but got 999
```

So, `low` is correctly set to 3, but `g:a1` (and thus `<line1>`) is 2 for some reason, although the first listed (and loaded) buffer is 3.